rector.php

```diff
@@ -38,5 +39,11 @@
 
     $containerConfigurator->import(LevelSetList::UP_TO_PHP_80);
 
-    $parameters->set(Option::SKIP, ['*/var/cache/*', __DIR__ . '/packages/Demo/data/DemoFile.php']);
+    $parameters->set(Option::SKIP, [
+        '*/var/cache/*',
+        __DIR__ . '/packages/Demo/data/DemoFile.php',
```

Comment:
@TomasVotruba the `DemoFile.php` is on root `./data' directory, do you think it can be removed from the skip option list as the file is not exists in the packages dir?

Reply:
I removed it from the skip config list of ecs, phpstan, and rector.php as file moved to root `data` directory https://github.com/rectorphp/getrector.org/pull/441/commits/55dddf42c495b3106f29d98b5510c9f3390922c4

Reply:
Ok :+1:

Reply:
@TomasVotruba it seems there is 500 error at demo page now after merge

![Screen Shot 2021-11-07 at 18 27 37](https://user-images.githubusercontent.com/459648/140643023-dbf1826e-89dd-48db-bb8c-17c5bcdcd772.png)

I will revert it to see if that related to it.